json/src/main/scala/akka/stream/alpakka/json/JsonParsing.scala

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright (C) 2016-2018 Lightbend Inc. <http://www.lightbend.com>
+ */
+
+package akka.stream.alpakka.json
+
+import scala.collection.immutable.Queue
+import akka.NotUsed
+import akka.stream.scaladsl.Flow
+import akka.stream.stage.{GraphStage, GraphStageLogic, InHandler, OutHandler}
+import akka.stream._
+import akka.util.ByteString
+import org.jsfr.json.compiler.JsonPathCompiler
+import org.jsfr.json.exception.JsonSurfingException
+import org.jsfr.json.path.JsonPath
+import org.jsfr.json.{JsonPathListener, JsonSurferJackson, ParsingContext}
+
+object JsonParsing {
```

Comment:
I think it should be `JsonSelect`, `JsonPathSelect` or something like that. The contained Json is still to be parsed - a design decision I like.

Reply:
Hmm, `JsonSelect.select` sounds a bit weird? What about `JsonReader`?